### PR TITLE
Add uploader to LegendasDivx subtitle results

### PR DESCRIPTION
**Pull request by redglory**

### CODE REVIEW

File: libs/subliminal_patch/providers/legendasdivx.py [hunk 1]
```diff
@@ -204,14 +202,20 @@ def _process_page(self, video, bsoup, querytext, videoname):
                 logger.warning('skipping subbox on %s' % self.searchurl.format(query=querytext))
                 continue
 
+            # get subtitle uploader
+            sub_header = _subbox.find("div", {"class" :"sub_header"}) 
+            uploader = sub_header.find("a").text if sub_header else '<n/a>'
```
Comment (morpheus65535) [inline libs/subliminal_patch/providers/legendasdivx.py:207]: else 'anonymous'

**Comment by morpheus65535** [comment]
Only one small comment but the rest seems pretty good. Thanks!

File: libs/subliminal_patch/providers/legendasdivx.py (hunk 1 above)
Comment (morpheus65535) [reply libs/subliminal_patch/providers/legendasdivx.py:222]: Please make this change and I'll be able to merge your PR. Good work BTW!

**Comment by redglory** [comment]
I've fixed this in 8b04941a3f71f49b661205f687c005a0b5f96d08

And I've fixed some things on the previous pull request 2dcfc433c3cfddb75a8c9678c60b2814feb78dfb

- now it searches for next pages if present
- fixed some exceptions which don't make sense  (fixes issue #962) and refined others
- improved some log debug messages

**Comment by morpheus65535** [comment]
> I've fixed this in [8b04941](https://github.com/morpheus65535/bazarr/commit/8b04941a3f71f49b661205f687c005a0b5f96d08)
> 
> And I've fixed some things on the previous pull request [2dcfc43](https://github.com/morpheus65535/bazarr/commit/2dcfc433c3cfddb75a8c9678c60b2814feb78dfb)
> 
> * now it searches for next pages if present
> * fixed some exceptions which don't make sense  (fixes issue #962) and refined others
> * improved some log debug messages

Thanks for doing this!

**Pull request merged**